config/initializers/ostatus.rb

```diff
@@ -12,7 +12,7 @@
   config.x.use_s3       = ENV['S3_ENABLED'] == 'true'
 
   config.action_mailer.default_url_options = { host: web_host, protocol: https ? 'https://' : 'http://', trailing_slash: false }
-  config.x.streaming_api_base_url          = 'http://localhost:4000'
+  config.x.streaming_api_base_url          = "http://localhost:#{port.to_i + 100}"
```

Comment:
Aha, I see. I think it might be better to hardcode PORT=4000 in this case because everyone's already used to it running on that port, and changing defaults will have lots of people's installations breaking.